ur_robot_driver/doc/features.md

```diff
@@ -1,32 +1,25 @@
-| trajectory extrapolation on robot on missing packages | no<sup>2</sup>   | yes                   |
-| use ROS as drop-in for TP-programs                    | -                | yes   |
-| extract calibration from robot                        | -                | yes                   |
-| send custom script commands to robot                  | yes              | yes               |
-| ROS 2 support                                         | ?                | (planned)<sup>3</sup> |
-| Reconnect on a disconnected robot                     | yes              | yes           |
+| Feature                                               | ROS2 Driver
+| ---                                                   | ---                       |
+| position-based control                                | yes                       |
+| scaled position-based control                         | yes                       |
+| velocity-based control                                | no<sup>1</sup>            |
+| reporting of tcp wrench                               | not yet implemented       |
+| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| pausing of programs                                   | yes                       |
+| continue trajectories after EM-Stop resume            | yes                       |
+| continue trajectories after protective stop           | yes                       |
+| panel interaction in between possible                 | yes                       |
+| get and set IO states                                 | yes                       |
+| use tool communication on e-series                    | not yet tested            |
+| use the driver without a teach pendant necessary      | yes                       |
+| support of CB2 robots                                 | -                         |
+| trajectory extrapolation on robot on missing packages | yes                       |
+| use ROS as drop-in for TP-programs                    | yes                       |
+| headless mode                                         | not yet implemented       |
+| extract calibration from robot                        | not yet implemented       |
+| send custom script commands to robot                  | yes                       |
+| Reconnect on a disconnected robot                     | yes                       |
 
-<sup>1</sup> Depending on the mode the driver is running the panel won't react or using the panel
-will stop the program without notifying the ROS user.
-
-<sup>2</sup> In velocity mode this is implicitly given.
-
-<sup>3</sup> There is no specific plan to do this inside of the first driver development. However,
-it is structured in a way so that a ROS2 driver should be developed as easy as possible by keeping
-as much as possible in a ros-independent library.
+<sup>1</sup> Currently not possible in ROS2 control. The driver will be extended as soon as the velocity controller is available
```

Comment:
Will suggest you add a version to the ROS2 control and remove the promise. Thus it is the intention.